.github/workflows/linux.yml

```diff
@@ -30,28 +31,35 @@ jobs:
     - name: Install Ubuntu dependencies
       run: |
         # NOTE: not the same OpenBLAS version as in upstream CI (I'm being lazy here)
-        sudo apt-get install libopenblas-dev libatlas-base-dev liblapack-dev gfortran libgmp-dev libmpfr-dev libsuitesparse-dev ccache libmpc-dev
+        sudo apt-get update
+        sudo apt-get -yq -o Debug::pkgProblemResolver=true -o Debug::Acquire::http=true install libopenblas-dev libatlas-base-dev liblapack-dev gfortran libgmp-dev libmpfr-dev libsuitesparse-dev ccache libmpc-dev
 
     - name: Install Python packages
       run: |
         python -m pip install numpy setuptools wheel cython pytest pytest-xdist pybind11 pytest-xdist mpmath gmpy2 pythran ninja
-        python -m pip install --user git+https://github.com/rgommers/meson.git@fix-cython-mixed-sources
+        python -m pip install git+https://github.com/rgommers/meson.git@fix-cython-mixed-sources
 
     - name: Build SciPy
       run: |
         meson setup build --prefix=$PWD/installdir
         ninja -C build -j 2
 
+    - name: Cache build dir
+      id: cache-scipy-build
+      uses: actions/cache@v2
+      with:
+        path: build
+        key: ${{ runner.os }}-${{ matrix.python-version }}
+
     - name: Install SciPy
       run: |
         meson install -C build
 
     - name: Test SciPy
       run: |
-        export PYTHONPATH=$PWD/installdir/lib/python3.9/site-packages/
+        export PYTHONPATH=$INSTALL_DIR
```

Comment:
This env var doesn't seem to be defined anywhere.

Reply:
Fixed now!